app/models/concerns/node_shared.rb

```diff
@@ -53,6 +53,23 @@ def self.graph_grid(body, _page = 1)
     end
   end
 
+  def self.simple_data_grapher(body, _page=1)
```

Comment:
You can do away with this and reuse the method from line 39-54.

Reply:
Actually, this is a different function for handling the new power tag of simple-data-grapher.  I don't think it will be a good idea to combine it with another function, can we ignore this warning?